ext/devfile-java8.yaml

```diff
@@ -1,10 +1,10 @@
 schemaVersion: 2.0.0
 metadata:
   name: java-openliberty-java8
-  version: 0.1.2
+  version: 0.3.0
```

Comment:
hmm.. this doesn't have to be versioned together with the parent..but seems fine to do..maybe better.